### PR TITLE
Check artwork versions

### PR DESCRIPTION
**Pull request by saolsen**
Adds version id to line items.

### CODE REVIEW

File: app/services/create_order_service.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def self.with_artwork!(user_id:, artwork_id:, edition_set_id: nil, quantity:)
       )
       order.line_items.create!(
         artwork_id: artwork_id,
+        artwork_version_id: artwork[:current_version_id],
```
Comment (williardx) [inline app/services/create_order_service.rb]: Should this ever be `nil`? Should we not create orders if there's no version id?

Comment (ashkan18) [reply app/services/create_order_service.rb]: i think we did migrate all artworks to have `current_version_id` but we could add a check here.

**Comment by ashkan18** [comment]
@saolsen looks like there is a `schema.rb` conflict, for resolving that the easiest approach i found locally was to merge `master` and accept incoming changes and then do a
```shell
rails db:drop
rails db:create
rails db:migrate
```

**Comment by saolsen** [comment]
Sorry, forgot about this. I merged master in and regenerated db. Should be good to go.

**Comment by ashkan18** [comment]
🙌

**Pull request merged**